### PR TITLE
docs(security) ingress auth using zone token

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
should we use a common term for zone ingress / zone egress?

zone proxy maybe?

### CODE REVIEW

File: docs/docs/dev/sidebar.json [hunk 1]
```diff
@@ -87,9 +87,9 @@
       "security/api-access-control",
       "security/api-server-auth",
       "security/dp-auth",
-      "security/zone-ingress-auth",
-      "security/zoneegress-auth",
-      "security/dp-membership"
+      "security/zoneproxy-auth",
+      "security/dp-membership",
+      "security/zone-ingress-auth"
```
Comment (lahabana) [inline docs/docs/dev/sidebar.json:92]: Do you want to get rid of the zone-ingress-auth page completely? You can setup redirects from zone-ingress-auth and zoneegress-auth to zoneproxy-auth and then just mention that zone-ingress-auth old way is deprecated (or inline the old doc in the new doc)

**Comment by lahabana** [comment]
@jakubdyszkiewicz what's next for this PR?

**Comment by lahabana** [comment]
@jakubdyszkiewicz seems like you need to update this PR

File: docs/docs/dev/sidebar.json (hunk 1 above)
Comment (jakubdyszkiewicz) [reply docs/docs/dev/sidebar.json]: I removed the page and setup a redirect.

**Comment by jakubdyszkiewicz** [comment]
refreshed for the jekyll rework

**Review by lahabana — APPROVED** [review]

**Review by lahabana — APPROVED** [review]

**Pull request merged**